test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -330,23 +298,25 @@ void test_all()
             count_set(1, 2), settings);
     }
 
-    /*** TODO: self-tangencies for difference
```

Comment:
This TODO was really old and the tests succeed now.
The 0,1 output is invalid because the input is invalid, and therefore it can be ignored.